### PR TITLE
Sentry manager methods

### PR DESCRIPTION
**Pull request by blfpd**
1. Register ModelAdmin classes with the decorator
1. Move some code from views to the QuerySet manager of SentryIssue

### CODE REVIEW

File: zoo/auditing/admin.py [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 from . import models
 
+admin.register(models.Issue)
+
```
Comment (aexvir) [inline zoo/auditing/admin.py]: 🤔 Hmm... shouldn't this also be using the decorator?

```suggestion
@admin.register(models.Issue)
```

Comment (aexvir) [reply zoo/auditing/admin.py]: oh well... GitHub seems to ignore the empty lines on the suggestions... 😅

Comment (blfpd) [reply zoo/auditing/admin.py]: Ups, yep, it seems I missed that one

**Review by aexvir — APPROVED** [review]
Thanks a lot! 🙂️

**Pull request merged**